inventory.schema.json

```
@@ -2788,6 +2788,10 @@
             "firmware": {
               "type": "string"
             },
+            "ip": {
+              "type": "string",
+              "title": "Remote device IP address"
+            },
```

Maybe should that be called `management_ip`? And also, since this is a new parameter, it's probably better to expect a correct format:
```suggestion
            "ip": {
              "type": "string",
              "oneOf": [
                  { "format": "ipv4" },
                  { "format": "ipv6" }
              ],
              "title": "Remote device IP address"
            },
```

---

This is not a new parameter. It is still and only provided by XML from netinventory task to tell on which ip a device was inventoried.
Probably the title should explain this.

---

I've checked, `IP` does exists in original specs, so we cannot enforce format, we may have issues with old data :/